src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -1249,12 +1249,12 @@ public IOpDelegationStrategy GetOpDelegationStrategy(UnaryOp op)
             return new DefaultUnaryOpDelegationStrategy(op, this);
         }
 
-        public ICallNodeDelegatableNodeValidationStrategy GetCallNodeDelegationStrategy()
+        public virtual ICallNodeDelegatableNodeValidationStrategy GetCallNodeDelegationStrategy()
         {
             return new DelegationValidationStrategy(this);
         }
 
-        public IDottedNameNodeDelegatableNodeValidationStrategy GetDottedNameNodeDelegationStrategy()
+        public virtual IDottedNameNodeDelegatableNodeValidationStrategy GetDottedNameNodeDelegationStrategy()
```

Comment:
why is DottedName delegation strategy something on TexlFunction at all?

Reply:
great question, I forgot to clean up having made this virtual, I don't think the getter is ever used though so it may be safe to remove altogether.

Reply:
It seems to be used in a lot of places: 
![image](https://user-images.githubusercontent.com/93291567/226489098-0975257f-fb8d-4222-8946-27bf465abed3.png)